RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -10,6 +10,8 @@
 #include "RecoTracker/LSTCore/interface/SegmentsSoA.h"
 #include "RecoTracker/LSTCore/interface/TripletsSoA.h"
 
+#include "Quintuplet.h"
```

Comment:
Is this include correct? I noticed the includes above it are all to the SoA headers. Figured I would ask to confirm.

Reply:
This is because `runDeltaBetaIterations` is defined there. At some point we should refactor it to move common functions into a separate header.